patches/minecraft/net/minecraft/item/ItemArmor.java.patch

```diff
@@ -1,6 +1,26 @@
 --- ../src-base/minecraft/net/minecraft/item/ItemArmor.java
 +++ ../src-work/minecraft/net/minecraft/item/ItemArmor.java
-@@ -221,6 +221,19 @@
+@@ -30,6 +30,8 @@
+ import net.minecraftforge.fml.relauncher.Side;
+ import net.minecraftforge.fml.relauncher.SideOnly;
+ 
++import javax.annotation.Nullable;
++
```

Comment:
Imports are not allowed in patches, you can write out `@javax.annotation.Nullable` where you need to use it.